src/client/components/sidebar/sidebar.js

```diff
@@ -29,8 +30,19 @@ class Sidebar extends React.Component {
 
   render() {
     let dealtCard = getDealtCard(this.props.G);
+    let all = new Set(getPlayers(this.props.ctx.numPlayers));
+    let passed = new Set(this.props.G.passed);
+    let difference = new Set([...all].filter(x => !passed.has(x)));
+    let whopassed = resolvePlayerNames(Array.from(difference), this.props.names, this.props.playerID);
+    let btncolor = "secondary", warningtxt = "invisible";
+
+    if (whopassed[0] === "You" && whopassed[1] === undefined) {
```

Comment:
I'm not sure I understand what is happening here:  If `You` are the only one who passed, then show the warning that you are last one to pass? Maybe the variable names need a bit of refinement?